cluster/juju/layers/kubernetes-worker/templates/default-http-backend.yaml

```diff
@@ -1,15 +1,23 @@
-apiVersion: v1
-kind: ReplicationController
+apiVersion: extensions/v1beta1
+kind: Deployment
 metadata:
   name: default-http-backend
+  labels:
+    app.kubernetes.io/name: default-http-backend
+    app.kubernetes.io/part-of: ingress-nginx
+    cdk-{{ juju_application }}-ingress: "true"
+  namespace: ingress-{{ juju_application }}-nginx
 spec:
   replicas: 1
   selector:
-    app: default-http-backend
+    matchLabels:
+      app.kubernetes.io/name: default-http-backend
+      app.kubernetes.io/part-of: ingress-nginx
```

Comment:
The concern about `{{ juju_application }}` missing from selector labels ([context](https://github.com/juju-solutions/kubernetes/pull/170/files#r223096924)) applies here, too.